### PR TITLE
Create Events Table

### LINKED ISSUE #7: Create Events Table
## Description
- Your task is to create some of the tables that we will be using throughout the rest of this project. 

## Instructions
- Create `events` table according to the database model
  - Refer to PostgreSQL documentation for creating a table
- Create `events` table schema under `server/db/schema/`.
- Name the schema file as `events.sql`

## Notes
- Arrows on the DB model indicate foreign constraints
- For now, assume that all attributes are `NOT NULL` unless otherwise specified
- Use DB Fiddle or Local Postgres to test when finished (NOT NEONDB PRODUCTION DATABASE)
- If your table relies on another table by foreign key, create a dummy table containing the primary key and any other fields needed

## Acceptance Criteria
- SQL creation code should exist in a file located in `server/db/schema/events.sql`
- Should be able to insert rows into table successfully (be sure to include pictures of this)

## Resources
- [DB Model](https://www.figma.com/board/n2WrXKm7kz13meVmRrwwEe/%5BCSE%5D-1C.-Database-Model?node-id=0-1&t=YlnByuTylGPBgJ3K-1) 
- [Create Table Tutorial](https://neon.tech/postgresql/postgresql-tutorial/postgresql-create-table)
- Refer to [Sample Table Creation](https://github.com/ctc-uci/cse/blob/main/server/db/schema/sample.sql)

### PR DESCRIPTION
**Pull request by angeliinawang**
## Description
Completed SQL code to create the Events table.

## Screenshots/Media
Screenshot of successful insertion of rows into Events table.
image

## Issues
Closes #7

### CODE REVIEW

**Comment by angeliinawang** [comment]
Screenshot of successful insert of rows into Events table
InsertiontoEvents

File: server/db/schema/events.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    location VARCHAR(256) NOT NULL,
+    title VARCHAR(256) NOT NULL,
+    description TEXT,
+    level LEVEL NOT NULL,
+    date DATE NOT NULL,
+    start_time TIME NOT NULL,
+    end_time TIME NOT NULL,
+    call_time TIME NOT NULL,
+    class_id INTEGER NOT NULL,
+    FOREIGN KEY (class_id) REFERENCES Classes(id),
```
Comment (stevem-zhou) [inline server/db/schema/events.sql]: We also want to include cascade on delete, in the event that the row that we are referencing gets deleted, then we can delete this row as well.

This is more of a NIT but you should also just list out all the columns first before adding in any constraints.

**Comment by stevem-zhou** [comment]
Could you guys also check out the cse-dev chat on slack and make sure that you guys have all the steps completed in this PR.

File: server/db/schema/events.sql [hunk 2]
```diff
@@ -0,0 +1,24 @@
+-- DROP TABLE IF EXISTS Events;
+-- DROP TABLE IF EXISTS Classes;
+
+DROP TYPE IF EXISTS level;
```
Comment (h0ethan04) [inline server/db/schema/events.sql]: Can you change from `level` to `LEVEL` for the consistency with docs?

File: server/db/schema/events.sql [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+CREATE TYPE level AS ENUM ('beginner', 'intermediate', 'advanced');
+
+-- CREATE TABLE Classes (
+--     id SERIAL PRIMARY KEY
+-- );
```
Comment (h0ethan04) [inline server/db/schema/events.sql]: Can you remove this from the pr?

**Review by h0ethan04 — CHANGES_REQUESTED** [review]

**Review by h0ethan04 — APPROVED** [review]
lgtm!

**Pull request merged**